components/camera/client.go

```diff
@@ -136,6 +137,43 @@ func (c *client) Stream(
 	return stream, nil
 }
 
+func (c *client) Images(ctx context.Context) ([]image.Image, time.Time, error) {
+	ctx, span := trace.StartSpan(ctx, "camera::client::Images")
+	defer span.End()
+
+	ctx, getSpan := trace.StartSpan(ctx, "camera::client::Images::GetImages")
+	resp, err := c.client.GetImages(ctx, &pb.GetImagesRequest{
+		Name: c.name,
+	})
+	getSpan.End()
```

Comment:
[nit] This won't be called if `GetImages` panics. You could wrap these few lines in an anonymous func.

Reply:
done